_includes/metrics_widgets/bubble-chart.html

```diff
@@ -0,0 +1,4 @@
+<div class="metrics__card {{class}}">
+    <canvas id="datagov-bubble-chart" class="datagov-charts" data-metric="{{metric}}"></canvas>
+</div>
+
```

Comment:
Leaving this for future use of bubble charts?

Reply:
I can take this out.